Makefile

```diff
@@ -209,7 +209,7 @@ $(ROM): $(ELF)
 	$(ELF2ROM) -cic 6105 $< $@
 
 $(ROMC): uncompressed
-	python3 tools/z64compress_wrapper.py --mb 32 --matching --threads $(N_THREADS) $(ROM) $@ $(ELF) build/$(SPEC)
+	python3 tools/z64compress_wrapper.py --matching --threads $(N_THREADS) $(ROM) $@ $(ELF) build/$(SPEC)
```

Comment:
Would we want to remove the `matching` flag for `NON-MATCHING` builds? I don't know enough about `z64compress` to know what differences it would make.

Reply:
All that removing this option does is allow z64compress to cut some corners to run a little faster, such as not padding the rom with the pattern `00010203...FF...` and skip some unnecessary alignment of rom segments. I don't mind adding something to not use this flag if `NON_MATCHING` is set, it would probably be nice to have.